assets/scss/components/_multiple-choice.scss

```diff
@@ -105,12 +105,10 @@
     }
 
     &::after {
-      top: calc(50% - 20px + 11px);
-      left: 9px;
-      width: 18px;
-      height: 7px;
-      -webkit-transform: rotate(-45deg);
-      -ms-transform: rotate(-45deg);
```

Comment:
We don’t need these transform lines anymore?

Reply:
We have autoprefixer now so that -webkit-transform, -ms-transform etc... should happen automatically

https://github.com/cds-snc/node-starter-app/blob/7813ffd6e1c4a86cf44d842655619b60db6ce683/postcss.config.js#L17